pkg/drain/evictor/pod.go

```diff
@@ -29,41 +29,41 @@ import (
 const (
 	daemonSetFatal      = "DaemonSet-managed Pods (use --ignore-daemonsets to ignore)"
 	daemonSetWarning    = "ignoring DaemonSet-managed Pods"
-	localStorageFatal   = "Pods with local storage (use --delete-local-data to override)"
+	localStorageFatal   = "Pods with local storage (use --Delete-local-data to override)"
```

Comment:
I think this slipped in

Reply:
Goland rename variable leaking into some places it shouldn't, my bad!